gateways/db/repository/account.go

```diff
@@ -6,28 +6,28 @@ import (
 	"log"
 	"reflect"
 
-	"github.com/francisleide/ChallangeGo/domain/entities"
+	"github.com/francisleide/ChallengeGo/domain/entities"
 )
 
 var (
-	id         string
-	nome       string
-	cpf        string
-	secret     string
-	balance    float64
-	created_at string
+	ID        string
+	name      string
+	CPF       string
+	secret    string
+	balance   float64
+	createdAt string
 )
 var accounts []entities.Account
 
-func (r Repository) List_all_accounts() []entities.Account {
+func (r Repository) ListAllAccounts() []entities.Account {
 
-	rows, err := r.Db.Query("SELECT id, nome, cpf, secret,balance, created_at from account;")
+	rows, err := r.Db.Query("SELECT id, name, cpf, secret,balance, created_at from account;")
 	defer rows.Close()
 	checkError(err)
 	fmt.Println("Reading data:")
 	for rows.Next() {
-		err = rows.Scan(&id, &nome, &cpf, &secret, &balance, &created_at)
-		account := entities.Account{id, nome, cpf, secret, balance, created_at}
+		err = rows.Scan(&ID, &name, &CPF, &secret, &balance, &createdAt)
+		account := entities.Account{ID, name, CPF, secret, balance, createdAt}
```

Comment:
```suggestion
var account entities.Account
err = rows.Scan(&account.ID, &account.Name, &account.CPF, &account.Secret, &accountBalance, &account.CreatedAt)
```
why won't you go like this? with this solution you will escape deadlock's as well